src/mpn_extras.h

```diff
@@ -121,26 +121,26 @@ flint_mpn_get_d(mp_srcptr ptr, mp_size_t size, mp_size_t sign, long exp);
 /* General multiplication ****************************************************/
 
 #ifdef FLINT_HAVE_FFT_SMALL
-#define FLINT_FFT_MUL_THRESHOLD 400
-#define FLINT_FFT_SQR_THRESHOLD 800
+# define FLINT_FFT_MUL_THRESHOLD FLINT_FFT_SMALL_THRESHOLD
+# define FLINT_FFT_SQR_THRESHOLD (2 * FLINT_FFT_SMALL_THRESHOLD)
```

Comment:
Is the squaring threshold really 2x on Intel too?

Reply:
Unfortunately it is exactly 2x on my machine... We should really try to look into if we can improve this.